src/libs/actions/Welcome/index.ts

```diff
@@ -204,9 +205,30 @@ function setSelfTourViewed(shouldUpdateOnyxDataOnlyLocally = false) {
     API.write(WRITE_COMMANDS.SELF_TOUR_VIEWED, null, {optimisticData});
 }
 
+function dismissProductTrainingElement(elementName: string) {
```

Comment:
Just a suggestion to keep the naming more consistent. Instead of sometimes having "element" and sometimes not. 
```suggestion
function dismissProductTraining(elementName: string) {
```